src/tactic.ts

```diff
@@ -5,7 +5,7 @@ enum TokenType {
     bracket,
     comment,
     string,
-    separator,
+    semicolon,  // ';' for the list separator
```

Comment:
Without asking I renamed this to semicolon because I thought 'semicolon' is really the simplest name :)